andela/casablanca-rc
# #157626915 Add popup confirmation before any Wallet transaction

#### What does this PR do?
Add popup confirmation before any transaction
Add a button to toggle payment form in the checkout page
#### Description of Task to be completed?
Add popup confirmation before any transaction
#### How should this be manually tested?
- clone the repo and change into the project directory
- start the application(instructions in the readme)
- visit the application in the browser 
- login/signup as a user
- visit your profile
- try transferring some fund to another customer
- a pop-up should show before the actual transaction
- try to also make payment with your wallet
- a pop-up should show before the actual transaction
- on the checkout page
- the payment option should display as button, 
- when you click on them, the actual payment form should be displayed
#### What are the relevant pivotal tracker stories?
#157626915
#### Screenshots 
<img width="1143" alt="screen shot 2018-05-18 at 9 17 10 am" src="https://user-images.githubusercontent.com/26048536/40223898-76da5d5e-5a7c-11e8-9796-5e931805ba6c.png">
<img width="1177" alt="screen shot 2018-05-18 at 9 17 38 am" src="https://user-images.githubusercontent.com/26048536/40223929-88fdea46-5a7c-11e8-9b96-614f417604cf.png">
<img width="524" alt="screen shot 2018-05-18 at 9 17 49 am" src="https://user-images.githubusercontent.com/26048536/40223944-917d3014-5a7c-11e8-8666-35ba45c5b909.png">
<img width="527" alt="screen shot 2018-05-18 at 9 21 58 am" src="https://user-images.githubusercontent.com/26048536/40224089-ffd3355e-5a7c-11e8-8d11-a5da13da8514.png">
